app/helpers/phone_format_helper.rb

```diff
@@ -34,7 +34,7 @@ def phone_microdata_for(type)
   # @return [String] The first voice or hotline phone number
   #   or nil if none are found.
   def first_voice_or_hotline_phone_for(phones)
-    phones.select { |phone| voice_or_hotline?(phone) }.first
+    phones.find { |phone| voice_or_hotline?(phone) }
```

Comment:
I don't have a strong preference either way, but I think `find` sounds better, and it's shorter. It's also the default Rubocop preference, so I think we'll go with that. I'll update our Hound config.